server/src/main/kotlin/suwayomi/tachidesk/manga/impl/extension/Extension.kt

```diff
@@ -23,7 +25,22 @@ import java.io.File
 object Extension {
     private val logger = KotlinLogging.logger {}
 
-    suspend fun installAPK(tmpDir: File, fetcher: suspend () -> File): Pair<String, List<CatalogueSource>> {
+    fun isNsfw(source: CatalogueSource): Boolean {
```

Comment:
Don't you also need to handle the case where a `SourceFactory` is annotated so everything created by it is also considered NSFW?

Reply:
Didn't even know that was possible, shouldn't be difficult to add

Reply:
Done